react/components/molecules/search/search.js

```diff
@@ -0,0 +1,155 @@
+import React, { PureComponent } from "react";
+import { Animated, StyleSheet, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+
+import { ButtonIcon, TextArea } from "../../atoms";
+
+export class Search extends PureComponent {
+    static get propTypes() {
+        return {
+            value: PropTypes.string,
+            placeholder: PropTypes.string,
+            buttonProps: PropTypes.object,
+            onFocus: PropTypes.func,
+            onValue: PropTypes.func,

```

Comment:
This is nitpicking, but maybe move onValue above onFocus, so that onFocus and onBlur stay together, since they are opposites